scripts/util.js

```diff
@@ -189,12 +189,35 @@ export function phoneFormat(num) {
   return phoneNum;
 }
 
+export const getDesign = (designType, defaultDesign, design1, design2, design3, design4, design5) => {
```

Comment:
This case statement seems very strange. Take the value and switch on a constant, but return one of the other parameters?